client/hawc_client/__init__.py

```diff
@@ -255,6 +285,48 @@ def references(self, assessment_id: int) -> pd.DataFrame:
         response_json = self.session.get(url).json()
         return pd.DataFrame(response_json)
 
+    def update_reference(self, reference_id: int, **kwargs) -> Dict:
+        """
+        Updates reference with given values. Fields not passed as parameters
+        are unchanged.
+
+        Args:
+            reference_id (int): ID of reference to update
+            **kwargs (optional): Named parameters of fields to update in reference. Example parameters:
+                title (str): title of the reference
+                abstract (str): reference abstract
+                tags (List[int]): tag IDs to apply to reference;
+                    completely replaces current tags
+
+        Example Usage:
+            updated_reference_json = client.lit.update_reference(
+                reference_id = 1,
+                title = "reference",
+                tags = [1,2,3]
```

Comment:
can you add if this appends or replaces the tags? I think I'd prefer replace, but we should be explicit here

Reply:
and if it's append by default, then we don't currently have a way to delete the tags I think, is that right?

Reply:
it replaces; I will add a note about that